participants/jonan.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Jonan Scheffler",
+    "company": "DXOS",
+    "when": {
+        "friday": true,
```

Friday spots are all taken! If you want to participate on Saturday, please change this to `false` and keep `saturday` set to `true`.

```suggestion
        "friday": false,
```

Once you change this, I can merge the PR and you're admitted for tomorrow.